jwst/jump/tests/test_detect_jumps.py

```diff
@@ -215,6 +234,51 @@ def test_multiple_neighbor_jumps_firstlastbad(setup_inputs):
     assert_array_equal(out_model.groupdq[0,:,4,4], [1, 0, 0, 0, 0, 0, 0, 0, 0, 1])
 
 
+def test_nirspec_saturated_pix(setup_inputs):
```

Comment:
This is the actual new test. All other changes are just cleanup and cosmetics (and some pep8 fixes).